### ISSUE TITLE
Strange issue not able to access inside of JSON for images?

### ISSUE THREAD

**Issue by KyleTryon** [contentful/blog-in-5-minutes#31]
I wonder if maybe you have run into anything like this? I have followed your tutorial and example to create my project but I am having a very strange issue.

When I get the JSON object back for my post, I am not able to access anything inside of the image object.
error: https://imgur.com/F35pMlQ

Full issue description:
https://github.com/KyleTryon/VidFireBlog/issues/1

**Comment by stefanjudis** [comment]
@KyleTryon Heyo. 👋🏻

I read the issue you have and it's really hard to follow your thoughts. 

But to give you a little bit of background: [This](https://www.contentful.com/developers/docs/concepts/links/#json-response) is how the contentful response looks like you're dealing with. It includes an `items` array for the fetched entries and an `includes` array which holds all the linked entries and assets.

The JavaScript SDK has link resolution built in for you so that you don't have to deal with that yourself. So there is some "magic" involved.

It might be that you're running into this somehow.

I set up a simple example and it works fine here 👉🏻 https://codepen.io/stefanjudis/pen/e45b05825accb618b68de991bb1cc6d4/. 

In your case, I believe it makes sense to encapsulate the problem from the surrounding code base?

**Comment by stefanjudis** [comment]
Closing due to inactivity.
